src/components/calcite-shell/usage/advanced.md

```diff
@@ -0,0 +1,115 @@
+#### Advanced
+
+Renders a shell with leading and trailing floating panels, action bar/pad, block, flow, tip manager, footer
+
+```html
+<calcite-shell>
+  <calcite-shell-panel slot="primary-panel" layout="leading" detached>
+    <calcite-action-pad id="action-pad" placement="side" slot="action-pad" hidden>
+      <calcite-action-group>
+        <calcite-action text="Add">
+          <!-- icon -->
+        </calcite-action>
+      </calcite-action-group>
+    </calcite-action-pad>
+    <calcite-action-bar slot="action-bar" theme="dark">
+      <calcite-action-group>
+        <calcite-action text="Add">
+          <!-- icon -->
+        </calcite-action>
+        <calcite-action text="Save" disabled>
+          <!-- icon -->
+        </calcite-action>
+        <calcite-action text="Layers" active indicator>
+          <!-- icon -->
+        </calcite-action>
+      </calcite-action-group>
+      <calcite-action-group>
+        <calcite-action text="Add">
+          <!-- icon -->
+        </calcite-action>
+        <calcite-action text="Layers" indicator>
+          <!-- icon -->
+        </calcite-action>
+      </calcite-action-group>
+    </calcite-action-bar>
+    <calcite-block collapsible  heading="Primary Content" summary="This is the primary.">
+      <calcite-block-content>
+        <calcite-action text="Side ActionPad" text-enabled id="action-pad-button" indicator>
```

Comment:
Maybe w can change the text from `side actionpad` to something more general so it doesn't appear like it should work already.

Same with `Anchored shell floating panel` and `Over shell floating panel`